js-packages/@prestojs/ui-antd/src/widgets/DateRangeWidget.tsx

```diff
@@ -20,8 +20,8 @@ function DateRangeWidget(
     ref: React.RefObject<DatePickerProps>
 ): React.ReactElement {
     const { lowerInput, upperInput, separator, ...rest } = props;
-    const { format: formatLower = 'MMMM Do YYYY', ...restLower } = lowerInput || {};
-    const { format: formatUpper = 'MMMM Do YYYY', ...restUpper } = upperInput || {};
+    const { format: formatLower = 'YYYY-MM-DD', ...restLower } = lowerInput || {};
+    const { format: formatUpper = 'YYYY-MM-DD', ...restUpper } = upperInput || {};
```

Comment:
i don't think we want to do this - this is shown to the user and YYYY-MM-DD is much less friendly/familiar than what was there